spec/support/helpers/form_helpers.rb

```diff
@@ -0,0 +1,31 @@
+module FormHelpers
+  def select_option(css_selector, value)
+    find(:css, css_selector).find(:option, value).select_option
+  end
+
+  def fill_in_player_data(index, player_name:, deck_name:, result:)
+    select_option("#game_game_records_attributes_#{index}_player_id", player_name)
+    select_option("#game_game_records_attributes_#{index}_deck_id", deck_name)
+    select_option("#game_game_records_attributes_#{index}_result", result)
+  end
+
+  def game_state(state)
+    select_option('#game_state', state.to_s)
+  end
+
+  def game_played_at(date)
```

Comment:
Here i don't understand what that method does by just reading the name. Let's use something more descriptive / in-line with the capybara DSL:
```rb
fill_in_game_played_at Date.current
```
Now that I'm thinking about it, it seems overkill to extract this method. If you compare the call sites it doesn't make it much more readable than the underlying logic:
```rb
fill_in_game_played_at Date.current

fill_in 'game_played_at', with: date
```
In which case let's consider removing this method (or renaming it if you still think it's an improvement).

Reply:
Same here not sure why that comment was hidden.